docker/dev.yml

```diff
@@ -51,7 +51,7 @@ services:
       - parabol-network
     restart: unless-stopped
   redis:
-    image: bitnami/redis:6.2
+    image: bitnami/redis:7.0-debian-11
```

Comment:
-1 I don't see why we should use the debian image locally when we can get away with the smaller alpine everywhere else. I don't quite understand the TLS argument either as we're not using TLS locally in any of our environments.

Reply:
We could just rollback to the basic image using `7.0-alpine` but if ever we need to test or use the TLS feature, we would need to adapt the configuration for it. See [this](https://github.com/ParabolInc/parabol/pull/8488#issuecomment-1701209184) and [this](https://github.com/ParabolInc/parabol/pull/8488#issuecomment-1701546727).

That said, I think you are right and I'm changing it to redis:7.0-alpine. If we ever need to test that again, we will see how to do so.